src/ui/Facet/Facet.ts

```diff
@@ -383,6 +384,18 @@ export class Facet extends Component {
      */
     numberOfValuesInFacetSearch: ComponentOptions.buildNumberOption({ defaultValue: 15, min: 1, section: 'FacetSearch' }),
 
+    /**
+     * Specifies whether [wildcards]{@link AllowedValuesPatternType.wildcards} will be used as the [allowedValuesPatternType]{@link IGroupByRequest.allowedValuesPatternType}
+     * in the [groupBy]{@link IGroupByRequest} for the facet search.
+     *
+     * Enabling this option will correctly return results for search values that match at the end of a field value.
```

Comment:
I think the wording can be improved here. I would also use a positive rather than a negation in the example. Here is my attempt:

```
Enabling this option will return results that terminate with the chosen field value.
For example, searching for "veo" will match "Coveo" when this option is enabled.
```
@dms1lva Should it match on 'field value' or 'search query' more generally?
 
@amoreauCoveo , what do you think?

Reply:
This is specifically for facet search, so it matches on 'field value'.

I used the negative here to match the situation that the user is experiencing by default. Using the positive might be better here, I'm not sure.

Reply:
Enabling this option returns results that end with the entered value. For example, For example, searching for `veo` would match with `Coveo`.